doc/specs-v2-frontend.md

```diff
@@ -0,0 +1,248 @@
+# Emails v2 : Frontend Specs
+
+This document's about the refactoring of the frontend of the Cozy-Email app. By the time, we saw some glitches in our current v1 version (see the [legacy](#legacy) part of the document) which results to an inconsistent state of app, causing it complex to maintain, test, and evolve without introducing new issues.
+
+We're strongly convinced that we need to rethink the base architecture of the app to stuck to a robust socle on which we can restart to add features and fix bugs in plain confidence. However, even if we probably need to rethink Ux paths too, we can keep current components by splitting and re-organizing them (see [components](#components) below), and keep a good part of the layouts.
+
+
+## App structure
+
+Application structure needs to drive how components interacts each with others to ensure separation of concerns is done right.
+
+```txt
```

Comment:
@m4dz maybe you can use [ditaa](http://ditaa.sourceforge.net/) to generate an image from this ascii-art. Personally, I prefer the ascii-art but it looks like I'm the minority :smile: